tests/helpers/setup-pretender.ts

```diff
@@ -0,0 +1,12 @@
+import Pretender from 'pretender';
+
+export default function setupPretender(hooks) {
```

Comment:
can you submit this as a dedicated PR? I guess we can merge that independent from the decorators discussion.

Reply:
Done 👍